threejs/src/index.js

```diff
@@ -4,13 +4,14 @@ import { fetchNFT, fetchActiveAd, sendMetric } from '../../utils/networking'
 export default class ZestyAd extends THREE.Mesh {
   /**
    * @constructor
-   * @param {Number} width Width of the ad, defaults to 2
-   * @param {Number} height Height of the ad, defaults to 3
    * @param {string} adSpace The adSpace ID
    * @param {string} creator The wallet ID of the creator
+   * @param {string} defaultAd The format of the default ad, defaults to square
+   * @param {Number} width Width of the ad, defaults to 2
+   * @param {Number} height Height of the ad, defaults to 3
    * @param {THREE.WebGLRenderer} renderer Optional field to pass in the WebGLRenderer in a WebXR project
    */
-  constructor(width = 2, height = 3, adSpace, creator, renderer = null) {
+  constructor(adSpace, creator, defaultAd = 'square', width = 2, height = 3, renderer = null) {
```

Comment:
`defaultAd` should be `adFormat` and `width` should be implicit. (Also this is not 3:4, does it still display correctly?)

Reply:
yeah i'm actually not sure why I had it set to that. probably just had a brainfart and meant to put 3 and 4 instead